### PR TITLE
fix(mep): Add outliers and satisfaction for lcp metrics [INGEST-1518]

### PR DESCRIPTION
**Pull request by untitaker**
We realized that the product also queries histograms for LCP and FCP, and that satisfaction metrics are also produced based on LCP.

Since LCP and duration can fall in different ranges wrt their p75 and p25, it means we need separate histogram outlier tags for each. So a transaction is an outlier/not an outlier depending on whether it is being queried by LCP, FCP or duration.

This is not true for satisfaction: There is exactly one way to calculate and query satisfaction for each transaction name, so we tag all relevant metrics by the same criteria.

### CODE REVIEW

**Comment by untitaker** [comment]
This change is insufficient:

* We need to add FCP for histograms
* We need to re-run the query for histogram outliers to compute preset values for FCP and LCP

@wmak please confirm that FCP and LCP use the same formula for determining outliers (p25 and p25 + 3 * (p75 - p25))

**Comment by wmak** [comment]
The outlier fence is calculated by `p75 + 3*abs(p75-p25)`

**Comment by untitaker** [comment]
>The outlier fence is calculated by p75 + 3*abs(p75-p25)

this says `**p25** + 3 * abs(p75 - p25)`: https://github.com/getsentry/sentry/blob/f796e9a93fcaa64d3ed58cfbc3a343666be36649/src/sentry/snuba/discover.py#L1053

**Comment by jjbayer** [comment]
> this says `**p25** + 3 * abs(p75 - p25)`, not `**p75** + 3 * abs(p75 - p25)`:
> 
> https://github.com/getsentry/sentry/blob/f796e9a93fcaa64d3ed58cfbc3a343666be36649/src/sentry/snuba/discover.py#L1155

@markus `third_quartile` is `p75` right? Not `p25`? `p75` would also match definition at https://en.wikipedia.org/wiki/Outlier#Tukey's_fences.

**Comment by untitaker** [comment]
ugh, yeah

File: src/sentry/relay/config/metric_extraction.py [hunk 1]
```diff
@@ -38,11 +39,16 @@ class MetricConditionalTaggingRule(TypedDict):
 _SATISFACTION_TARGET_METRICS = (
     "s:transactions/user@none",
     "d:transactions/duration@millisecond",
+    "d:transactions/measurements.lcp@millisecond",
```
Comment (jjbayer) [inline src/sentry/relay/config/metric_extraction.py:42]: Out of curiosity, why are we adding this?

**Review by jjbayer — APPROVED** [review]
Logic looks sound to me. A PR description would be nice.

File: src/sentry/relay/config/metric_extraction.py (hunk 1 above)
Comment (wmak) [reply src/sentry/relay/config/metric_extraction.py:42]: I believe we need this to fix apdex for projects that calculate it based on LCP

**Comment by untitaker** [comment]
@wmak if you have time, please verify locally that this fixes the bugs, otherwise I'll just merge and we'll see...

**Comment by wmak** [comment]
@untitaker I'll give it a try :+1:

**Comment by wmak** [comment]
@untitaker lgtm, I loaded several events normally and got a histogram then stored two outlier events and the histograms still looked reasonable. Hopefully that carries over to prod

**Pull request merged**